api/src/main/java/edu/wpi/first/shuffleboard/api/util/NetworkTableUtils.java

```diff
@@ -30,8 +30,8 @@ private NetworkTableUtils() {
   }
 
   @FunctionalInterface
-  public interface ITableListenerEx {
-    void valueChangedEx(ITable source, String key, Object value, int flags);
+  public interface NetworkTableListenerEx {
```

Comment:
This interface can be removed entirely now that we don't need to wrap the listener to get the flags